app/src/main/res/layout/item_monster_list.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/card_margin"
+    android:foreground="?android:attr/selectableItemBackground"
+    card_view:cardElevation="@dimen/card_elevation">
```

Comment:
Generally, `app` is used for XML namespace other than `android`

Reply:
Fix it.